spec/unit/puppet/provider/ios_snmp_trap/cisco_ios_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+module Puppet::Provider::IosSnmpTrap; end
+require 'puppet/provider/ios_snmp_trap/cisco_ios'
+
+RSpec.describe Puppet::Provider::IosSnmpTrap::CiscoIos do
+  let(:utility) { described_class }
+
+  def self.load_test_data
+    PuppetX::CiscoIOS::Utility.load_yaml(File.expand_path(__dir__) + '/test_data.yaml', false)
+  end
+
+  it_behaves_like 'resources parsed from cli'
+  it_behaves_like 'commands created from instance'
+  # it_behaves_like 'a noop canonicalizer'
+
+  describe 'convert_to_boolean' do
+    it 'given value' do
+      input = 'cake'
+      output = true
+      expect(utility.convert_to_boolean(input)).to eq output
+    end
+    it 'given nil' do
+      input = nil
+      output = false
+      expect(utility.convert_to_boolean(input)).to eq output
+    end
```

Comment:
These two could be simplified too:

```
it { expect(utility.convert_to_boolean('cake')).to eq true }
it { expect(utility.convert_to_boolean(nil)).to eq false }
```

As the `it 'given *` is implied by the expect.

Same with the `it`'s in the `false_to_unset` test cases.

Reply:
have simplified, kept 
```
it 'given string' do
      expect(utility.false_to_unset('cake')).to eq 'cake'
end
```
Kept it in the it do end as I prefer each test to have a descriptor